webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -87,6 +89,14 @@ export const LinkTooltip = ({href, connected, show, theme}) => {
         let date = new Date(data.created_at);
         date = date.toDateString();
 
+        let description = '';
+        if (data.body) {
+            description = data.body.substring(0, maxTicketDescriptionLength).trim();
+            if (description.length > maxTicketDescriptionLength) {
+                description += '...';
+            }
+        }
```

Comment:
I actually got this wrong in my previous suggestion. `description.length` will never be greater than `maxTicketDescriptionLength` in that block

```suggestion
        let description = '';
        if (data.body) {
            description = data.body.substring(0, maxTicketDescriptionLength).trim();
            if (data.body.length > maxTicketDescriptionLength) {
                description += '...';
            }
        }
```